examples/script_nifti.py

```diff
@@ -0,0 +1,41 @@
+import os
+import pathlib
+import shutil
+import sys
+from shimmingtoolbox.dicom_to_nifti import dicom_to_nifti
+import tempfile
+import urllib.request as ur
+from zipfile import ZipFile
+
+
+def main():
+    scriptPath = pathlib.Path(__file__).parent.absolute()  # Folder where this script is located
+    dataPath = os.path.join(scriptPath, 'data_testing')  # Folder where the data will be downloaded
+
+    # Download data when not already present
+    if not os.path.isdir(dataPath):
+        url = 'https://github.com/shimming-toolbox/data-testing/archive/r20200713.zip'
+        try:
+            fname_archive = 'data-testing.zip'
+            with ur.urlopen(url) as resp, open(fname_archive, 'wb') as out_file:
```

Comment:
We should think about adding some sort of checksumming on this. Something in the vein of how arch (or really any distro) packages things: https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/fbida#n18

```
source=(https://www.kraxel.org/releases/${pkgname}/${pkgname}-${pkgver}.tar.gz{,.asc}
        ida-desktop.patch)
sha1sums=('fe2ec92dc98296e46cd8ddddcd8b3a169cb506a0'
          'SKIP'
          'd06b70f2d581a1a3e9e0909b5af7f3f71ed54717')
```

we cannot trust HTTP to give us valid data.

Reply:
so we have to verify it